### PR TITLE
Re-enable multiple relations

### PR DESCRIPTION
**Pull request by hatched**
This only allows multiple relations to be created for ghost services. Once a service is deployed there is a further bug down the line which prevents multiple relations from being formed.

### CODE REVIEW

**Comment by hatched** [comment]
#### To QA

File: jujugui/static/gui/src/app/views/topology/relation.js [hunk 1]
```diff
@@ -1139,12 +1103,13 @@ YUI.add('juju-topology-relation', function(Y) {
       // Set up the relation data.
       var endpoint1 = endpoints[0][0],
           endpoint2 = endpoints[1][0];
-      var relation_id = 'pending-' + endpoint1 + endpoint2;
+      var idBlock = `${endpoint1}${endpoint2}`;
```
Comment (hatched) [inline jujugui/static/gui/src/app/views/topology/relation.js]: these 3 line modifications was the actual fix, everything else was just dead code removal.

File: jujugui/static/gui/src/test/test_topology_relation.js [hunk 2]
```diff
@@ -383,7 +337,7 @@ describe('topology relation module', function() {
     // Ensure the given relation includes the expected fields.
     var assertRelation = function(relation) {
       assert.strictEqual(
-          relation.get('relation_id'), 'pending-service1service2');
+          relation.get('relation_id'), 'pending-service1service2dbdb');
```
Comment (hatched) [inline jujugui/static/gui/src/test/test_topology_relation.js]: test for the code modification.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3827/
Test PASSed.

**Comment by huwshimi** [comment]
QA OK, nicely done @hatched, thanks for removing all that old code :+1:

**Comment by hatched** [comment]
Thanks for the review! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**